conf/nginx/nginx.conf.web.https.default.template

```diff
@@ -58,6 +58,14 @@ server
         set $login_upstream    ${web.upstream.webclient.target};
     }
 
+    if ($arg_loginOp = "logout") {
```

Comment:
should we merge these two and above condition?

Reply:
yeah that's what the initial plan was, but apparently nginx conf does not support or condition, it could be achieved using regex, but was giving some issues when Shashi tried that out.